revive.toml

```diff
@@ -20,6 +19,8 @@ warningCode = 0
 [rule.modifies-parameter]
 
 # Add these once issues are fixed
+[rule.blank-imports]
```

Comment:
This should be fixed in upstream https://github.com/kubernetes-sigs/apiserver-builder-alpha/pull/443